src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -359,6 +372,17 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
+		}
+
+        internal void InitializeNamespacesToExcludeFromLibrary()
+        {
+            if (!NamespacesToExcludeFromLibrarySpecified)
+            {
+                NamespacesToExcludeFromLibrary.Add(
+                    "ProtoGeometry.dll:Autodesk.DesignScript.Geometry.TSpline"
```

Comment:
If we remove the entry from Preferences.xml, will this property start showing up? How are you dealing with backward compatibility, I mean when user has migrated from older version to this one?